sdk/core/azure-core-http-vertx/pom.xml

```diff
@@ -0,0 +1,175 @@
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.7.0</version> <!-- {x-version-update;com.azure:azure-client-sdk-parent;current} -->
+    <relativePath>../../parents/azure-client-sdk-parent</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-core-http-vertx</artifactId>
+  <packaging>jar</packaging>
+  <version>1.9.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core-http-vertx;current} -->
```

Comment:
version should be "1.0.0-betat.1".  You might need to run 

python eng\versioning\update_versions.py --update-type library --build-type client

https://github.com/Azure/azure-sdk-for-java/blob/main/eng/versioning/update_versions.py#L14

Reply:
Version updated as requested.